spec/views/idv/address/new_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe 'idv/address/new' do
```

Comment:
Thanks for adding a view test! Other files of this type seem to be named `new.html.erb_spec.rb`. I thought `new_spec.rb` was a leftover placeholder at first.

Reply:
Good catch. Changed.